WearOAuth/oauth-pkce/src/main/java/com/example/android/wearable/oauth/pkce/AuthPKCEActivity.kt

```diff
@@ -32,24 +30,24 @@ private const val TAG = "WearOAuthActivity"
  * The sample uses the Google OAuth 2.0 APIs, but can be easily extended for any other
  * OAuth 2.0 provider.
  *
- * See [WearOAuthViewModel] for the implementation of the OAuth flow.
+ * See [AuthPKCEViewModel] for the implementation of the OAuth flow.
  */
-class WearOAuthActivity : ComponentActivity() {
+class AuthPKCEActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.layout)
+        setContentView(R.layout.activity_auth)
         // In a real world situation you would use some form of Dependency Injection here.
-        val viewModel = WearOAuthViewModel(RemoteAuthClient.create(applicationContext))
+        val viewModel = AuthPKCEViewModel(RemoteAuthClient.create(applicationContext))
```

Comment:
Ditto here for using `by viewModels` so that the `viewModelScope` will be cleaned up appropriately.

Reply:
Fixed!